src/coreclr/crossgen-corelib.proj

```diff
@@ -45,6 +45,10 @@
       <CrossGen2DllFiles Condition="'$(CrossDir)' != ''" Include="$(BinDir)/$(CrossDir)/crossgen2/*" />
     </ItemGroup>
 
+    <ItemGroup>
+      <OptimizationMibcFiles Include="$(MibcOptimizationDataDir)/$(TargetOS)/$(TargetArchitecture)/**/*.mibc" />
```

Comment:
Do we want to produce some warnings when no MIBC data is available? Or, alternatively, should we print the files that were found or at least the number of files found? Especially for lab runs this might simplify various investigations.

Reply:
The build log will contain the list of mibc files processed as part of the standard output from the dotnet-pgo tool. Also, it will fail if there isn't any input data.